book/website/communication/personal/personal-website.md

```diff
@@ -25,6 +25,56 @@ But this space is not necessarily limited to academic content; it can also be a
 As a result, this website can be customised to express the individual's unique identity and values.
 This online presence can help researchers expand their visibility, engage with a wider audience, connect with potential collaborators, and even create new career opportunities.
 
+(cm-personal-website-basics)=
+# Basic elements of Web Development: Processes and Tools
+
+Web development is the entire process of creating and maintaining websites and web applications.
+
+It involves iterative ideation and implementation. The core phases associated with web development are analysis, design, development, testing & review, deployment, and maintenance.
+
+(cm-personal-website-analysis)=
+## Analysis 
+
+The goal of the analysis phase is to gain a deep understanding of the problem that a website is trying to solve so that the best solution can be found. By taking the time to carefully analyse the requirements of a website, people (researcher, web developer, programmer, and more) can ensure that the final product is effective and meets its users' needs.
```

Comment:
We try to follow [semantic line breaks](https://sembr.org/) in the book to help keep diffs clean.